lib/uff_db_loader.rb

```diff
@@ -80,17 +84,41 @@ def self.databases
     end
   end
 
-  def self.replace_database_name_in_config(new_database_name)
+  def self.database_name_template(old_database_name)
+    "<%= UffDbLoader.current_database_name || '#{old_database_name}' %>"
+  end
+
+  def self.setup_dynamic_database_name_in_config
     old_database_name = Rails.configuration.database_configuration["development"]["database"]
 
     return false if old_database_name.nil?
 
     old_config = File.read(UffDbLoader.config.database_config_file)
-    new_config = old_config.sub(old_database_name, new_database_name)
+    new_config = old_config.sub(old_database_name, database_name_template(old_database_name))
     File.write(UffDbLoader.config.database_config_file, new_config)
   end
 
+  def self.current_database_name
+    database_name = File.read(database_name_file).strip
+    return if database_name.blank?
+    database_name
```

Comment:
Würde das nicht das selbe machen?

```suggestion
    File.read(database_name_file).strip
```

Reply:
`""` is `blank?`, but not falsy.

```rb
"" || "foo" # => ""
```